clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -125,6 +125,9 @@ Improvements to clang-tidy
 - Added argument `--exclude-header-filter` and config option `ExcludeHeaderFilterRegex`
   to exclude headers from analysis via a RegEx.
 
+- Added argument `--allow-empty-checks` and config option `AllowEmptyCheckList`
```

Comment:
you need to document this also in other places in documentation.

Reply:
maybe better --allow-no-checks or something that would suggest that clang-tidy will not fail with empty check list